rust/pact_verifier_cli/Cargo.toml

```diff
@@ -49,3 +49,11 @@ expectest = "0.12.0"
 rstest = "0.24.0"
 tempfile = "3.20.0"
 trycmd = "0.15.9"
+
+
+[[bin]]
+name = "pact-verifier"
+path = "src/bin.rs"
+[lib]
+name = "pact_verifier_cli"
+path = "src/lib.rs"
```

Comment:
The path specification here is redundant. In fact, I think the whole section might be redundant? I was under the impression that if there's a `src/lib.rs`, then automatically Cargo defines a library, using the `package.name` value.